internal/services/mssql/mssql_elasticpool_resource_test.go

```diff
@@ -344,6 +342,8 @@ resource "azurerm_mssql_elasticpool" "test" {
   max_size_gb         = %.7[6]f
   zone_redundant      = %[9]t
 
+  maintenance_configuration_name = azurerm_resource_group.test.location == "westeurope" ? "SQL_WestEurope_DB_2" : "SQL_Default"
```

Comment:
Unfortunately, that won't work as this new field breaks `TestAccMsSqlElasticPool_basicDTU` with 
```
Specified maintenance configuration 'DB_2' doesn't exist or is not supported by service level objective 'B1M50
```

It looks like it's not supported because of the basic sku.